le_dash/settings/base.py

```diff
@@ -150,3 +150,14 @@
 }
 
 BANNER_BASE_URL = getenv('BANNER_BASE_URL')
+
+# Use local redis server to cache searches or template fragments,
+# timeout is 1 hour
+# Use KEY_PREFIX for production
+CACHES = {
```

Comment:
It takes a decorator to cache a view:
eg: @cache_page(60 * 60)
For template fragments you need something like:
{% load cache %}
{% cache 500 sidebar %}
    .. sidebar ..
{% endcache %}
I haven't set it up to do that yet. Next iteration perhaps.
More details from https://docs.djangoproject.com/en/1.10/topics/cache/